packages/safe-tools-client/app/components/future-payments-list/index.gts

```diff
@@ -0,0 +1,60 @@
+import Component from '@glimmer/component';
+import BoxelActionContainer from '@cardstack/boxel/components/boxel/action-container';
+import WalletService from '@cardstack/safe-tools-client/services/wallet';
+import NetworkService from '@cardstack/safe-tools-client/services/network';
+import HubAuthenticationService from '@cardstack/safe-tools-client/services/hub-authentication';
+import ScheduledPaymentsService from '@cardstack/safe-tools-client/services/scheduled-payments';
+import TokensService from '@cardstack/safe-tools-client/services/tokens';
+import { inject as service } from '@ember/service';
+import { on } from '@ember/modifier';
+import { action } from '@ember/object';
+import lte from 'ember-truth-helpers/helpers/lte';
+
+import './index.css';
+
+export default class FuturePaymentsList extends Component {
+  @service declare hubAuthentication: HubAuthenticationService;
+  @service declare network: NetworkService;
+  @service declare scheduledPayments: ScheduledPaymentsService;
+  @service declare tokens: TokensService;
+  @service declare wallet: WalletService;
+
+  get futurePayments() {
+    return [] //TODO: fetch future payments
+  }
+
+  @action
+  async addFund() {
+    //TODO: implement add fund action
+  }
+
+  <template>
+    <BoxelActionContainer 
+      class="future-payments-list"
+      as |Section ActionChin|>
+      {{#if (lte this.futurePayments.length 0)}}
+        <Section class="no-payments-section">
+          <div class="no-payments-title">Scheduled your first payment</div>
```

Comment:
Would be good to use more BEM-like classes here, for example:

Block: `future-payments-list`
Elements: `future-payments-list__no-payments-section`, `future-payments-list__no-payments-title`, `future-payments-list__no-payments-description`

Reply:
Then you can remove nested CSS selectors like `.no-payments-section .no-payments-title` and use `future-payments-list__no-payments-title` instead

Reply:
Updated